src/detect-engine-analyzer.c

```diff
@@ -1000,6 +1000,7 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
 
     SCJbSetString(ctx.js, "raw", s->sig_str);
     SCJbSetUint(ctx.js, "id", s->id);
+    SCJbSetUint(ctx.js, "iid", s->iid);
```

Comment:
Do we need this ?
If so, own commit ?

Reply:
I removed the label typo/doc update because of this line :-p

Reply:
Hmmm... Good question. Should we expose the internal id? Is there any value to this?